.gitignore

```diff
@@ -1,6 +1,6 @@
 .phpcs-cache
-composer.phar
```

Comment:
Why removing this rule?

Reply:
It is not part of this project and is rather user specific. 

It should be eventually [ignored globally](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

Reply:
maybe we should add this to the documentation? cause composer documentation push to use composer.phar direct in the project itself. but I agree this is no a be deal.